### PR TITLE
Yahoo finance stock data

### PR DESCRIPTION
**Pull request by apfong**
Shows some of the data we can get from standard stock quotes.

### CODE REVIEW

**Comment by chriskonstad** [comment]
Works on my machine :+1: 

Btw, I rebased on master for you to remove the extraneous commits.

**Comment by apfong** [comment]
Thanks man!

I tried my best :disappointed:

**Comment by bmwwoo** [comment]
Works for me. LGTM :mag:

File: Gemfile.lock [hunk 1]
```diff
@@ -176,6 +177,4 @@ DEPENDENCIES
   turbolinks
   uglifier (>= 1.3.0)
   web-console (~> 2.0)
-
```
Comment (Sakibs) [inline Gemfile.lock]: I noticed that this appears in some branches and in some it is not there. Is this linked to some gem maybe?

Comment (bmwwoo) [inline Gemfile.lock]: @Sakibs are you talking about the yahoofinance gem? All of this is auto generated code from the installed gems

File: app/assets/stylesheets/yahoo_finance.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.tt-hint {
+  color: #999;
+}
+
+.tt-menu {
+  width: 396px;
+  padding: 8px 0;
+  background-color: #fff;
```
Comment (bmwwoo) [inline app/assets/stylesheets/yahoo_finance.scss]: The front end guy in me says best practices are to have a _colors.scss file for every hex code and rgb value but we can do that separately.

File: app/lib/yahoo_finance.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      #   [1] - Open
+      #   [2] - High
+      #   [3] - Low
+      #   [4] - Close
+      #   [5] - Volume
+      #   [6] - Adjusted Close
+
+      days_past = 36500
+      hist = YahooFinance::get_historical_quotes_days(symbol, days_past)
+      histJSON = hist.map { |e| {:id => e}}.to_json
```
Comment (bmwwoo) [inline app/lib/yahoo_finance.rb]: `id: e` instead of hashrocket

File: app/views/yahoo_finance/stocks.html.erb [hunk 4]
```diff
@@ -0,0 +1,60 @@
+<h4>Enter stock ticker from NASDAQ:</h4>
+
+<%= form_tag({}, method: 'get', id: 'submit_container') do %>
+  <%= label_tag(:symbol, "Ticker:", class: 'symbol') %>
```
Comment (bmwwoo) [inline app/views/yahoo_finance/stocks.html.erb]: double quotes to single quotes

**Comment by bmwwoo** [comment]
:+1:

File: app/assets/stylesheets/yahoo_finance.scss (hunk 2 above)
Comment (apfong) [inline app/assets/stylesheets/yahoo_finance.scss]: Ok we can do that, i just took that from the sentiment page

File: app/lib/yahoo_finance.rb [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+      #   [3] - Low
+      #   [4] - Close
+      #   [5] - Volume
+      #   [6] - Adjusted Close
+
+      days_past = 36500
+      hist = YahooFinance::get_historical_quotes_days(symbol, days_past)
+      histJSON = hist.map { |e| {id: e}}.to_json
+
+      { hist: histJSON }
```
Comment (bmwwoo) [inline app/lib/yahoo_finance.rb]: @chriskonstad is talking about these hashes

**Pull request merged**